app/renderer/components/navbar_menu.js

```diff
@@ -1,9 +1,19 @@
 "use strict";
+const {
```

Comment:
This is not  very important, but in this project, by convention, [I don't use destructing for require statements](https://www.youtube.com/watch?v=6vxHkAQRQUQ)

So this statement should be `const shell=require('electron').shell` to have the same style as the rest of the code